app/assets/src/components/views/SampleView/SampleViewControls.jsx

```diff
@@ -27,7 +33,14 @@ class SampleViewControls extends React.Component {
     let v = pipelineRun && pipelineRun.pipeline_version;
     if (v) resParams["pipeline_version"] = v;
 
-    let res = `/samples/${sample.id}/report_csv`;
+    // Set the min contig size.
+    if (minContigSize) resParams["min_contig_size"] = minContigSize;
+
+    // Need to know what version of the report table we're looking at and request accordingly
+    let res =
+      reportVersion === 1
```

Comment:
Instead of using this parameter just use the feature flag that was already injected in the context.
Example of use in: https://github.com/chanzuckerberg/idseq-web/blob/390ae2d2a5b41220c601ebf72584900ecc842d03/app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx#L609
```
<UserContext.Consumer>
              {currentUser => 
	if (currentUser.allowedFeatures.includes(REPORT_PAGE_V2))
```
You need to add a new feature constant.

Reply:
Removed the parameter, changed to redirecting on the server-side.